IaC/AKS/cluster/charts/ngsa/values.yaml

```diff
@@ -0,0 +1,81 @@
+# Default values for ngsa.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: retaildevcrew
+  name: ngsa
+  tag: "beta"
+  pullPolicy: Always
+  
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+app:
+  args: [""]
```

Comment:
Are you still planning on adding the helm override file? Overriding the image values, app arguments as well as the Ingress host below will be useful.

Reply:
Yeah I was thinking the same, we can discuss in more detail during our meeting

Reply:
Done